cpanfile

```diff
@@ -4,16 +4,30 @@ use warnings;    # satisfy linter
 requires 'Code::TidyAll::Plugin::SortLines::Naturally';
 requires 'Code::TidyAll::Plugin::Test::Vars';
 requires 'Code::TidyAll::Plugin::UniqueLines';
+
 requires 'Devel::Cover';
 requires 'Devel::Cover::Report::Codecov';
 requires 'Devel::Cover::Report::Coveralls';
+
+# let's also add dzil to the party using ETHER flavor :-)
+requires 'Dist::Zilla::PluginBundle::Author::ETHER';
+
```

Comment:
Can we remove the newlines and put the comments on the same line with the `requires`?  I like to alpha sort the deps and this makes it easier.  :)

Reply:
sure